samples/ConsoleApp/ConsoleApp.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Please add targets `net452;net46;461`, these are also targeted by the SDK 1.0.1 and will be useful on Windows tests. You will need to add a conditional item group to the project, something like:

```xml
  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
    <Reference Include="System.Net.Http" />
  </ItemGroup>
```